src/litserve/mcp.py

```diff
@@ -20,18 +20,24 @@
 from contextlib import asynccontextmanager
 from typing import TYPE_CHECKING, Any, Dict, List, Optional, Union, get_args, get_origin
 
-import mcp.types as types
-from fastapi import FastAPI
-from mcp.server.fastmcp.server import _convert_to_content
-from mcp.server.lowlevel import Server as MCPServer
-from mcp.server.streamable_http_manager import StreamableHTTPSessionManager
 from pydantic import BaseModel
 from starlette.applications import Starlette
 from starlette.routing import Mount
 from starlette.types import Receive, Scope, Send
 
 from litserve.utils import is_package_installed
 
+if not is_package_installed("fastmcp"):
```

Comment:
should we rather call this on MCP init? just in case this path accidentially ends up on an import path somehow but isn't actually used.